packages/koenig-lexical/test/utils/background-color-helper.js

```diff
@@ -0,0 +1,28 @@
+export async function cardBackgroundColorSettings(page, {cardColorPickerTestId, customColor, colorTestId, findByColorTitle, imageUploadId, fireColorSetting = true}) {
+    if (fireColorSetting) {
+        const colorSetting = page.locator(`[data-testid="${cardColorPickerTestId}"]`);
+        const colorButton = colorSetting.locator('button');
+        await colorButton.click();
+    }
+
+    if (findByColorTitle) {
+        const colorTitle = page.locator(`[title="${findByColorTitle}"]`);
+        await colorTitle.click();
+    }
+
+    if (customColor) {
+        const picker = page.locator(`[data-testid="color-picker-toggle"]`);
+        await picker.click();
+        const colorInput = page.locator(`input[aria-label="Color value"]`);
+        await colorInput.click({clickCount: 3});
```

Comment:
Why does this need three clicks?

Code comments are really helpful for anything out of the ordinary like this to explain the why